### DISCUSSION TITLE
界面略显粗犷，精细一点就更好了

### DISCUSSION THREAD

**Discussion by cethink** [usememos/memos#1823] [General]
### Is your feature request related to a problem?

no

### Describe the solution you'd like

例如字号和图标可以小一点，统一一点，看着比较协调就可以了

[a comment by an automated account is not shown]

**Comment by 5kyfkr** [comment]
我觉得界面挺好，字体比flomo清晰多了，flomo那种低对比元素加小字体太费眼

[a comment by an automated account is not shown]

**Comment by GodMeowIceSun** [comment]
在考虑要不要完成两个功能

- [ ]  支持各框架元素控制的主题功能
- [ ]  基于类型标签的扩展展示和交互插件功能

[a comment by an automated account is not shown]
